archive/100-theorems-list/inverse-triangle-sum.lean

```diff
@@ -0,0 +1,87 @@
+import tactic
+import algebra.big_operators
+import data.real.basic
+
+lemma range_insert (n : ℕ) :
+insert n (finset.range n) = finset.range (n + 1) :=
+begin
+ext, rw finset.mem_range,
+simp only [finset.mem_insert, finset.mem_range],
+split; omega,
+end
+
+
+lemma discrete_FTC
+{α : Type*}
+[comm_ring α]
+(f F : ℕ → α)
+(hF : F 0 = 0)
+:
+(∀ n,(finset.range n).sum f = F n) ↔
+∀ n, F (n + 1) - F n = f n :=
```

Comment:
Put these assumptions on one line (max width 100 characters). The statement of the theorem, if it goes on a separate line, should be indented by two spaces, as should any excess hypotheses beyond the first line.